config/styleguide.config.json

```diff
@@ -51,6 +51,7 @@
       "Fieldset",
       "FloatingButton",
       "FloatingButtonGroup",
+      "GlobalNavigation",
```

Comment:
Please move `GlobalNavigation` to `systems/` instead`  - It's way more complex than just a component.

Reply:
I'd argue.

Aside of its size GlobalNav is still a single component with no additional requirements which accepts a bunch of props and does not care about the rest of app, while BannerNotification comes with store injection, actions and a dedicated model.

Reply:
I think that is a fair point but GlobalNav is more of an organism whereas components should be molecules/atoms. http://bradfrost.com/blog/post/atomic-web-design/#atoms

Reply:
At which moment a 'component' becomes an 'organism'/'system'? It's not clear for me, and could cause some issues with understanding the concept. Eg. if we'd like to move `OpenGraph` card to common, should it be a 'component' or an 'organism'?

My understanding of the file structure was described above - if you need to export something more than a component itself, it should become a system. But in other case, from dev/react perspective, a component (no matter how big) is still a encapsulated entity that you can use and pass props to.

Maybe to avoid such issues we should rename 'systems' to eg. 'redux' and keep there only related stuff?